src/test/scala/QuartzCalendarSpec.scala

```diff
@@ -169,7 +170,7 @@ class QuartzCalendarSpec extends Specification with ThrownExpectations { def is
           CronOnlyBusinessHours {
             type = Cron
             excludeExpression = "* * 0-7,18-23 ? * *"
-            timezone = "America/San_Francisco"
+            timezone = "America/Los_Angeles"
```

Comment:
Why were the timezones changed in these places?

Reply:
Test was not passing for me. I suspect my default timezone is different than "America/San_Francisco" so change the system default to a known timezone. The test error was:

[error] x Be able to parse and create a Holiday calendar
[error]  Set(Sat Mar 30 00:00:00 PDT 2013, Sat Apr 19 00:00:00 PDT 2014, Sat Apr 04 00:00:00 PDT 2015, Sat Mar 26 00:00:00 PDT 2016, Sat Apr 15 00:00:00 PDT 2017) does not contain Sun Mar 31 00:00:00 PDT 2013, Sun Apr 20 00:00:00 PDT 2014, Sun Apr 05 00:00:00 PDT 2015, Sun Mar 27 00:00:00 PDT 2016, Sun Apr 16 00:00:00 PDT 2017 but contains Sat Apr 15 00:00:00 PDT 2017, Sat Mar 26 00:00:00 PDT 2016, Sat Apr 04 00:00:00 PDT 2015, Sat Apr 19 00:00:00 PDT 2014, Sat Mar 30 00:00:00 PDT 2013 (QuartzCalendarSpec.scala:71)

Reply:
That's interesting. I just tried pulling that timezone locally and it doesn't really exist for me either (`TimeZone.getTimeZone("America/San_Francisco")` returns GMT. I think switching this to `America/LosAngeles` is actually correct. I'll take care of that. Thanks for finding this.